lib/active_model/serializable_resource.rb

```diff
@@ -9,6 +9,7 @@ def initialize(resource, options = {})
       @resource = resource
       @adapter_opts, @serializer_opts =
         options.partition { |k, _| ADAPTER_OPTION_KEYS.include? k }.map { |h| Hash[h] }
+      notify_active_support
```

Comment:
not sure if it's any simpler, but we already provide a logger, so you could set the logger to use a stringio and capture the output.

in any case, I'm unsure if this is the best place in this class for this method, but don't have a better idea yet